RevenueCatUI/Data/PaywallViewConfiguration.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  PaywallViewConfiguration.swift
+//
+//
+//  Created by Nacho Soto on 1/19/24.
+//
+
+import Foundation
+
+import RevenueCat
+
+/// Parameters needed to configure a ``PaywallView``.
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+struct PaywallViewConfiguration {
```

Comment:
In android, this would be similar to `PaywallOptions`, which is public and it's how developers can customize most things for the paywall. We used the builder pattern there, but I'm not sure if that's so common in swift.

Reply:
Yeah we could think about exposing this (and only the public types). We can do that in a separate PR.